index.html

```diff
@@ -9965,22 +10018,35 @@ <h4 id="Beast Shape - Small Animal">Beast Shape - Small Animal</h4>
             <tr>
                <td>Core
                </td>
+               <td>Ankle Bite
+               </td>
+               <td>Make a regular melee attack against a target. On a successful hit do
+                  Light+DEX poison damage.
+               </td>
+               <td>E, Melee
+               </td>
+               <td>2
+               </td>
+            </tr>
+            <tr>
+               <td>1
+               </td>
                <td>Poison Strike
                </td>
                <td>Make a regular melee attack against a target. On a successful hit do
-                  Medium+DEX poison damage and <a class="condition" data-tooltip-container>hobble</a>. 2 FORT to resist <a class="condition" data-tooltip-container>hobble</a>
+                  Light+DEX poison damage and <a class="condition" data-tooltip-container>paralyse</a>. 2 FORT to resist <a class="condition" data-tooltip-container>paralyse</a>
```

Comment:
Paralyse at level 1 is probably definitely too much. We've avoided this so far, it's just a bit too good, even if it does require shapeshifting into an ugly rat to do it first.